### PR TITLE
Key argument for dump binds command (closed #2619)

### LINKED ISSUE #2619: show bind if you just type bind <key>
Maybe it is possible to show binds when you just type bind and then comes the bind which is on the key right now
Like I bind on Q Kill and then I type bind q thats shows something like `bind q kill` or something else xd

### PR DESCRIPTION
**Pull request by ChillerDragon**
Inspired by ddnet code thanks to @marcelherd:
https://github.com/ddnet/ddnet/commit/1f5859503329717b3f633b9331b353db133ca021

```
> binds
[...]
[2020-05-06 13:58:02][binds]: mouse1 (411) = +fire
[2020-05-06 13:58:02][binds]: mouse2 (412) = +hook
[2020-05-06 13:58:02][binds]: mousewheelup (420) = +prevweapon
[2020-05-06 13:58:02][binds]: mousewheeldown (421) = +nextweapon
> bind q kill
[2020-05-06 13:58:05][binds]: bound q (113) = kill
> binds q
[2020-05-06 13:58:08][binds]: q (113) = kill
```

### CODE REVIEW

File: src/game/client/components/binds.cpp [hunk 1]
```diff
@@ -326,15 +326,44 @@ void CBinds::ConBinds(IConsole::IResult *pResult, void *pUserData)
...
-			if(pBinds->m_aaaKeyBindings[i][m][0] == 0)
-				continue;
-			str_format(aBuf, sizeof(aBuf), "%s%s (%d) = %s", GetModifierName(m), pBinds->Input()->KeyName(i), i, pBinds->m_aaaKeyBindings[i][m]);
+			str_format(aBuf, sizeof(aBuf), "key '%s' not found", pKeyName);
 			pBinds->Console()->Print(IConsole::OUTPUT_LEVEL_STANDARD, "binds", aBuf);
 		}
+		else
+		{
+			if(pBinds->m_aaaKeyBindings[KeyID][Modifier][0] == 0)
+				str_format(aBuf, sizeof(aBuf), "%s (%d) is not bound", pKeyName, KeyID);
```
Comment (oy) [inline src/game/client/components/binds.cpp:344]: Could you display the `modifier` here too like the other messages

Comment (ChillerDragon) [reply src/game/client/components/binds.cpp:344]: If I understand the code correctly ``pKeyName`` already includes the modifier.

Comment (oy) [reply src/game/client/components/binds.cpp:344]: Then you should do it like that for the else case 2 lines below

Comment (ChillerDragon) [reply src/game/client/components/binds.cpp:344]: Good idea. That made the long line shorter :)

What about in the 0 argument case?

**Pull request merged**